velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -466,6 +476,36 @@ TEST_F(ArithmeticTest, hexWithVarbinaryAndVarchar) {
   ASSERT_EQ(toHex("Spark\u6570\u636ESQL"), "537061726BE695B0E68DAE53514C");
 }
 
+TEST_F(ArithmeticTest, widthBucket) {
```

Comment:
Maybe add a test for min == max.

Reply:
already have one 
![image](https://github.com/facebookincubator/velox/assets/9858245/be04cfdf-2572-49d0-9ac2-a9316db1a1f2)